applications/ChimeraApplication/python_scripts/rotate_region_process.py

```diff
@@ -0,0 +1,95 @@
+import KratosMultiphysics 
+import KratosMultiphysics.ChimeraApplication as KratosChimera
+import warnings
+
+def Factory(settings, Model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return ApplyRotateRegionProcess(Model, settings["Parameters"])
+
+
+class ApplyRotateRegionProcess(KratosMultiphysics.Process):
```

Comment:
I guess it is not possible to separate Torque and ALE from this process?

Reply:
No really, What I can do is to make calculateTorqueProcess and use it inside here.